src/main/resources/templates/macros/json/output.vm

```diff
@@ -1,16 +1,21 @@
-#macro(includeOutput, $output)
+#macro(includeOutput, $outputs, $isPassed)
 
-#if ($output)
+#if($outputs)
   <div class="outputs inner-level">
-    #foreach($message in $output.getMessages())
+    #foreach($output in $outputs)
+      #set($outputId = $counter.next())
+      #set($index = $foreach.index + 1)
       <div class="output indention">
-        #set($outputId = $counter.next())
-        <div data-toggle="collapse" class="collapsable-control" data-target="#msg-$outputId">
-          #set($index = $foreach.index + 1)
-          <a>Output $index</a>
+        <div data-toggle="collapse" class="#if ($isPassed) collapsed #end collapsable-control" data-target="#output-$outputId">
+          Output $index
+          <i class="chevron fa fa-fw"></i>
```

Comment:
For collapsing functionality for outputs at the same style like steps. Please see screenshots above.